sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -32,7 +32,7 @@ public async Task RunSamplesAsync()
 
             string componentModelId = await GetUniqueModelIdAsync(SamplesConstants.TemporaryComponentModelPrefix, DigitalTwinsClient).ConfigureAwait(false);
             string modelId = await GetUniqueModelIdAsync(SamplesConstants.TemporaryModelPrefix, DigitalTwinsClient).ConfigureAwait(false);
-            string dtId = await GetUniqueTwinIdAsync(SamplesConstants.TemporaryTwinPrefix, DigitalTwinsClient).ConfigureAwait(false);
+            string dtId1 = await GetUniqueTwinIdAsync(SamplesConstants.TemporaryTwinPrefix, DigitalTwinsClient).ConfigureAwait(false);
```

Comment:
Something for us to consider in general - Instead of using numbering for Ids, it might be good to give examples with names. Ex: CampusDigitalTwin/CafeDigitalTwin etc.. 
Ex: If you see the documentation section of ADT for modelId, they try to use some sort of real world scenario - https://github.com/Azure/azure-digital-twins/blob/private-preview/Documentation/concepts-models.md

Reply:
I don't disagree, but this is a type that doesn't have a meaningful name.

I chose dt1 and dt2 because they are 2 instances of the same type, serialized 2 different ways.